### ISSUE TITLE
FTP "No Auto Power Off" is not working

### ISSUE THREAD

**Issue by 444nonymous** [aldostools/webMAN-MOD#52]
To reproduce:

1. Set "system auto-off" in the PS3 to 20 minutes and check "No Auto Power Off" in webman.
  (I also have set an FTP 10 minutes timeout in webman)
2. FTP to the PS3 and you can check that "system auto-off" values are now changed to OFF.

(So it should work. But It does not.)

3. Start transferring a game, better if it has many small files. Don't touch the PS3 controller.
4. After 20 minutes the PS3 is turned off, leaving many FTP files queued.
5. Turn on the PS3, and the FTP transfer automatically resumes.
6. This time the PS3 never turns off, because "system auto-off" settings are off and webman will keep them as default.

I'm not sure why it fails. But it always does.

**Comment by 444nonymous** [comment]
I was transfering GT6 today and It happens even in the middle of transfering the 14GB file "GT.VOL"

[12:27:13] STOR /dev_hdd0/GAMES/Gran Turismo 6/PS3_GAME/USRDIR/GT.VOL
[12:27:13] 150 OK
[12:44:05] QUIT
[12:44:05] Connecting to ps3.lan, port 21 (#1)

If I turn on the PS3, the transfer is auto resumed.

[12:48:03] Connected to server, waiting for response...
[12:48:03] 220-VSH ftpd
[12:48:03] 220 webMAN ftpd 1.46.01 MOD
[12:48:03] USER anonymous
[12:48:03] 331 OK
[12:48:03] PASS (hidden)
[12:48:03] 230 OK
[12:48:03] SYST
[12:48:03] 215 UNIX Type: L8
[12:48:03] CWD /dev_hdd0/GAMES/Gran Turismo 6/PS3_GAME/USRDIR
[12:48:03] 250 OK
[12:48:03] PWD
[12:48:03] 257 "/dev_hdd0/GAMES/Gran Turismo 6/PS3_GAME/USRDIR"
[12:48:03] TYPE I
[12:48:03] 200 TYPE OK
[12:48:03] PASV
[12:48:03] 227 Entering Passive Mode (10,20,30,107,164,231)
[12:48:03] REST 5570322432
[12:48:03] 350 REST command successful
[12:48:03] STOR /dev_hdd0/GAMES/Gran Turismo 6/PS3_GAME/USRDIR/GT.VOL
[12:48:03] 150 OK

**Comment by 444nonymous** [comment]
But it was resumed badly. The 14GB file was copyied as 10GB, with no error in the FTP transfer log. So it looked like the file transfer was successful in the FTP but it was not when I checked the HDD contents.

I had to delete it and transfer again. I also did a HDD filesystem check, because the new transfer was very slow 1mb/s. The HDD is new 7200 rpm. After the filecheck I got 8mb/s (@100MB LAN).

Did this corrupt the filesystem? (Trasnfering 14GB stored as 10GB). I don't know, the PS3 filesystem likes to hide free space and do weird things when not written properly.
